src/ViewLayer/DisplayDashboard/DisplayDashboardView/DisplayDashboardView.cpp

```diff
@@ -106,8 +106,8 @@ void DisplayDashboardView::connectMppt(MpptPresenter& mpptPresenter)
 {
     connect(&mpptPresenter, SIGNAL(mpptReceived(int, Mppt)),
             this, SLOT(mpptReceived(int, Mppt)));
-    connect(&mpptPresenter, SIGNAL(mpptPowerReceived(double)),
-            this, SLOT(mpptPowerReceived(double)));
+    //connect(&mpptPresenter, SIGNAL(mpptPowerReceived(double)),
```

Comment:
`mpptPowerReceived` isn't being used anywhere so let's remove all references to it.